### ISSUE TITLE
Action bar not working

### ISSUE THREAD

**Issue by melinstagibson** [TheLimeGlass/Skellett#7]
Hey, i tried the exact same code as in the documentation:

```
command /ab:
    permission: *
    trigger:
        show action bar with "askj" to the player
```

but it only throws:

 [Skript] Can't understand this condition/effect: show action bar with "askj" to the player (test.sk, line 4: show action bar with "askj" to the player')

im on spigot 1.11, latest build
Benskus skript for (lastes)
skrayfall 1.9.5
and you skquery build

*and skelett 1.6.9.4

**Comment by TheLimeGlass** [comment]
Issue is 1.11. will fix this in next update.

**Comment by TheLimeGlass** [comment]
Fixed in 1.6.9.7
